src/css/fonts.css

```diff
@@ -0,0 +1,41 @@
+@charset "UTF-8";
+
+@import url("https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,300i,400,400i,600,600i,700,700i,900,900i");
+@import url("https://fonts.googleapis.com/css?family=Source+Serif+Pro:400,600,700");
+@import url("https://fonts.googleapis.com/css?family=Roboto+Slab:300,400,700");
+@import url("https://fonts.googleapis.com/css?family=Kalam:300,400,700");
+@import url("https://fonts.googleapis.com/css?family=Noto+Sans:400,400i,700,700i&subset=devanagari");
+@import url("https://fonts.googleapis.com/css2?family=Source+Serif+Pro:ital,wght@0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap");
+
+/** Import Font Awesome Fonts */
+@font-face {
+  font-family: 'Font Awesome 5 Free';
```

Comment:
I have a feeling that we're going to need the FA solids also, now that they're used in the new alerts, but that could be in another PR.

Reply:
Good point. Let's pull those in.